app/domain/logs.rb

```diff
@@ -121,7 +121,7 @@ module ResourceRestrictions
 
       ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
         msg: "Extracting resource restrictions for authenticator '{0-authn-name}' from host '{1-host-name}'",
-        code: "CONJ00039D"
+        code: "CONJ00032D"
       )
 
```

Comment:
@hilagross
How can we insure the deleted code not appears somewhere in the documentation?

Reply:
@tzheleznyak We checked docsumentation it is not in there.
@shulifink please make sure it wasn't added lately to our docs.